.eslintrc.json

```diff
@@ -33,7 +33,9 @@
         "bracketSpacing": true,
         "arrowParens": "avoid"
       }
-    ]
+    ],
+    "import/no-unresolved": ["off"],
```

Comment:
Why this is necesary?

Reply:
## Without this line Eslint show me this error
![image](https://user-images.githubusercontent.com/61360270/166303738-fdd0c19a-1178-443e-b033-dc59cb0cf845.png)

Reply:
Hmm is it because that path doesn't exists? SHould be GraphQL << note the uppercase L

Reply:
I didn't note it, Thanks